### PR TITLE
wording patch to dbus hostname change warning

### PR DESCRIPTION
**Pull request by dt-rush**
Change-type: patch
Connects-to: #1271
Signed-off-by: dt-rush <nickp@balena.io>

### CODE REVIEW

**Comment by garethtdavies** [comment]
In addition, it seems this section is outdated and could do with a review https://www.balena.io/docs/learn/develop/runtime/#dbus-communication-with-host-os, in particular:

```In some cases its necessary to communicate with the host OS systemd to perform actions on the host, for example changing the hostname. To do this you can use dbus. ```

This seems redundant given it's immediately followed by:

```Changing the device hostname via a dbus-send method invocation ... is no longer possible```

If there is a specific version that made this incompatible we should mention it (if we want to keep the section about it being possible). I'm guessing Supervisor 1.7.0 is _really_ old and is that still necessary?

**Comment by garethtdavies** [comment]
@dt-rush I couldn't make a suggested change so for simplicity just pushed another commit removing the example of dbus hostname. That work for you?

File: pages/learn/develop/runtime.md [hunk 1]
```diff
@@ -54,7 +54,7 @@ root@raspberrypi3-cc723d7:/# printenv | grep {{ $names.company.allCaps }}
...
 
-In some cases its necessary to communicate with the host OS systemd to perform actions on the host, for example changing the hostname. To do this you can use [dbus][dbus-link]. In order to ensure that you are communicating to the host OS systemd and not the systemd in your container it is important to set `DBUS_SYSTEM_BUS_ADDRESS` for all dbus communication. The setting of that environment variable is different for older and newer devices (based on the {{ $names.company.lower }} supervisor version), choose the line that is correct for your device's OS version (can be found in your device dashboard):
+In some cases its necessary to communicate with the host OS systemd to perform actions on the host. To do this you can use [dbus][dbus-link]. In order to ensure that you are communicating to the host OS systemd and not the systemd in your container it is important to set `DBUS_SYSTEM_BUS_ADDRESS` for all dbus communication. The setting of that environment variable is different for older and newer devices (based on the {{ $names.company.lower }} supervisor version), choose the line that is correct for your device's OS version (can be found in your device dashboard):
```
Comment (lurch) [inline pages/learn/develop/runtime.md]: "some cases its necessary" -> "some cases it's necessary"

**Comment by garethtdavies** [comment]
@balena-ci retest

**Pull request merged**